### PR TITLE
Feature muboot

### PR DESCRIPTION
**Pull request by flbulgarelli**
See See https://github.com/mumuki/mumukit-platform/wiki/%5BMuBoot%5D-Standard-Mumuki-Platform-Boot-Process

### CODE REVIEW

File: devstart [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+echo ' |    |   \  || \_\ \  |_|  (  <_> )  | |   Y  \  ___/\  \___ / __ \_'
+echo ' |______  /__||___  /____/__|\____/|__| |___|  /\___  >\___  >____  /'
+echo '        \/        \/                         \/     \/     \/     \/ '
+echo ''
+
+echo "[MumukiDevstart::Classroom] Installing dependencies...."
+npm install
+
+echo "[MumukiDevstart::Classroom] Starting server...."
+npm start
```
Comment (faloi) [inline devstart]: As far as I know, this starts classroom in production mode, not development. The right command is `node_modules/.bin/gulp dev` (perhaps editing the `package.json` is a good idea).

File: devstart [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+echo ' |    |  _/  || __ \|  | |  |/  _ \   __\  |  \_/ __ \_/ ___\\__  \  '
+echo ' |    |   \  || \_\ \  |_|  (  <_> )  | |   Y  \  ___/\  \___ / __ \_'
+echo ' |______  /__||___  /____/__|\____/|__| |___|  /\___  >\___  >____  /'
+echo '        \/        \/                         \/     \/     \/     \/ '
+echo ''
+
+echo "[MumukiDevstart::Classroom] Installing dependencies...."
+npm install
+
+echo "[MumukiDevstart::Classroom] Starting server...."
```
Comment (faloi) [inline devstart]: Not classroom. :stuck_out_tongue:

File: devstart (hunk 1 above)
Comment (faloi) [inline devstart]: Ok, I was talking about classroom, but check it anyway.

Comment (flbulgarelli) [inline devstart]: Yes. I have asked @fedescarpa in that PR. I think there is something wrong with the gulp tasks.

File: devstart (hunk 2 above)
Comment (flbulgarelli) [inline devstart]: :stuck_out_tongue:

File: devstart (hunk 1 above)
Comment (fedescarpa) [inline devstart]: In bibliotheca it's OK but in classsroom... now it's ok

**Pull request merged**